File: cpp/1066.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    int even = 0, odd = 0, positive = 0, negative = 0, n, i;
    
    for (i = 0; i < 5;  i++) {
        cin >> n;

        if (n % 2 == 0) {
            even++;
        } else {
            odd++;
        }

        if (n > 0) {
            positive++;
        } else if (n < 0) {
            negative++;
        }
    }

    printf("%d valor(es) par(es)\n", even);
    printf("%d valor(es) impar(es)\n", odd);
    printf("%d valor(es) positivo(s)\n", positive);
    printf("%d valor(es) negativo(s)\n", negative);
    return 0;
}

File: py/1066.py
values = [int(input()) for i in range(5)]

even, odd, positive, negative = 0, 0, 0, 0
for i in values:
    if i % 2 == 0:
        even += 1
    else:
        odd += 1
        
    if i > 0:
        positive += 1
    elif i < 0:
        negative += 1

print(f'{even} valor(es) par(es)')
print(f'{odd} valor(es) impar(es)')
print(f'{positive} valor(es) positivo(s)')
print(f'{negative} valor(es) negativo(s)')